core/interfaces.go

```diff
@@ -220,6 +230,8 @@ type Storage interface {
 	GetMessage(orgID string, appID string, ID string) (*model.Message, error)
 	CreateMessageWithContext(ctx context.Context, message model.Message) (*model.Message, error)
 	UpdateMessage(message *model.Message) (*model.Message, error)
+	InsertRecipientsToMessage(recipients []model.MessageRecipient, messageID string) error
```

Comment:
It should be `InsertMessagesRecipients(recipients []model.MessageRecipient) error`